op-challenger/flags/flags.go

```diff
@@ -71,6 +71,11 @@ var (
 		EnvVars: prefixEnvVars("MAX_CONCURRENCY"),
 		Value:   uint(runtime.NumCPU()),
 	}
+	L2RpcFlag = &cli.StringFlag{
+		Name:    "l2-rpc",
```

Comment:
My suggestion would be to add:
```suggestion
	L2RpcFlag = &cli.StringFlag{
		Name:    "l2-rpc",
        Aliases: []string{"cannon-l2"},
```

and remove the `CannonL2Flag`. Means we don't have to manage the two separate flags.